webapp/src/Form/Type/ContestType.php

```diff
@@ -25,12 +25,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $this->addExternalIdField($builder, Contest::class);
         $builder->add('shortname', TextType::class);
-        $builder->add('name', TextType::class);
+        $builder->add('name', TextType::class, [
+            'help' => 'Contest name as shown in the top right.'
+        ]);
         $builder->add('activatetimeString', TextType::class, [
             'label' => 'Activate time',
+            'help' => 'Time when the contest becomes visible for teams to join. Must be in the past to enable submission of jury submissions.',
```

Comment:
Instead of:

> Time when the contest becomes visible for teams to join

isn't

> Time when the contest becomes visible for teams

Since there is no actual joining.